index.js

```diff
@@ -14,6 +15,21 @@ const baseURL = "https://www.icloud.com/shortcuts/api/records/";
  */
 const baseLink = "https://www.icloud.com/shortcuts/";
 
+/**
+ * A shortcut metadata.
+ */
+class ShortcutMetadata {
+	constructor(object) {
+		// Not sure if we keep the native Apple names for the properties?
```

Comment:
For these property names, I'd take off the `WFWorkflow` part and make it camel-cased, e.g. `WFWorkflowImportQuestions` would become `this.importQuestions`. For the client, it'd also probably be better to have `this.client.version` and `this.client.minimumVersion`.